### ISSUE TITLE
Datepicker updates only on hover

### ISSUE THREAD

**Issue by leider** [bootstrap-vue/bootstrap-vue#4858]
### Describe the bug

Changing v-model's value of a datepicker does not update the displayed date.

### Steps to reproduce the bug

1. Run the demo (code below)
2. Click on April; "Value" updates as expected, "Choose a date" does not.
3. Put mouse over the date's display
4. Update occurs

### Expected behavior

Displayed date syncs with the displayed value

### Versions
2.5.0

**Libraries:**

- BootstrapVue: 2.5.0
- Bootstrap: 4.4.1
- Vue: 2.6.11

**Environment:**

- Device: Mac
- OS: MacOS
- Browser: [Chrome; Safari]
- Version: [current]

### Demo link

```javascript
<template>
  <div id="app">
    <div class="row">
      <div class="col-3">
        <b-form-group label-for="example-datepicker" label="Choose a date">
          <b-form-datepicker id="example-datepicker" v-model="value" class="mb-2" locale="en-US"></b-form-datepicker>
        </b-form-group>
        <p>Value: '{{ value }}'</p>
        <b-button v-on:click="april">April</b-button>
        <b-button v-on:click="march">March</b-button>
      </div>
    </div>
  </div>
</template>

<script>

  export default {
    name: 'App',
    data() {
      return {
        value: '2020-03-01'
      }
    },
    methods: {
      april() {
        this.value = '2020-04-01';
      },

      march() {
        this.value = '2020-03-01';
      }
    }
  }
</script>

```
### Additional context

Same on Safari iOS, except hovering not working there obviously

**Comment by tmorehouse** [comment]
Duplicate: #4827

Will be fixed in v2.6

**Comment by leider** [comment]
Sorry. Did not find it as I was only looking for open issues. I’m an id**t
